electron/src/machines/aquapath/aquapath1/Aquapath1ControlPage.tsx

```diff
@@ -163,6 +182,56 @@ export function Aquapath1ControlPage() {
                   }
                 />
               </Label>
+
+              {backCurrentTemperature < backHeatingBoundary &&
+                backTargetFlow &&
+                (state?.flow_states.front.flow ?? 0) && (
+                  <Badge variant="default">
+                    Power: {back_power.current?.value} W<br />
+                    Total: {back_total_energy.current?.value} kWh
+                  </Badge>
+                )}
+              {backCurrentTemperature > backCoolingBoundary &&
```

Comment:
You might have an easier life if you'd just introduce helpers returning a boolean for code like that instead of repeatedly writing similar conditions all over the place.